src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
+    "transport": "http",
+    "icon":"http://google.tieto.com/images/services/google-cloud-platform/BigQuery_500px-1f2fdb56.png",
+    "clients": [
```

Comment:
I'm hesitant to structure the data to  support multi-clients at this point.  I'd be tempted to just keep it simple and minimal for now.